libr/asm/arch/msp430/msp430_disas.c

```diff
@@ -449,19 +432,27 @@ int msp430_decode_command(const ut8 *in, int len, struct msp430_cmd *cmd) {
 	ut8 opcode = get_twoop_opcode (instr);
```

Comment:
A question: does "int len" measure the length in bytes or in words (8 or 16 bits)?